src/legacy/js/functions/_loadEmbedIframe.js

```diff
@@ -15,6 +15,8 @@ function loadEmbedIframe(onSave) {
     function saveUrl() {
         var embedUrl = $('input#embed-url').val();
         var fullWidth = $('input#full-width-checkbox').is(':checked');
+        var embedUrl2 = $('input#embed-url-2').val();
+        console.log(embedUrl2," - 1");
```

Comment:
Oops, looks like there might be a rogue `console.log` left. Unless we think this is useful in the production environment it's probably worth deleting.